cypress_shared/helpers/apply.ts

```diff
@@ -313,7 +313,7 @@ export default class ApplyHelper {
 
   completeSentenceInformation() {
     // Given I click the sentence information task
-    Page.verifyOnPage(TaskListPage).clickTask('sentence-information')
+    Page.verifyOnPage(TaskListPage, this.application).clickTask('sentence-information')
```

Comment:
I'd have squashed this commit into the [previous one](https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/pull/530/commits/6270416f87a3b44e95c69c68054d45e2baa2840a) given that this commit looks like a fix to get the previous change working.